CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # MicroProfile Language Server Changelog
 
+## [0.13.0](https://github.com/eclipse/lsp4mp/milestone/14?closed=1) (October 17, 2024)
+
+### Enhancements
+
+- feat: Provide property validator extension API. See [#460](https://github.com/eclipse/lsp4mp/pull/460).
+
+### Bug Fixes
+
+- REST endpoint codelens is not resolving a `public static final` parameter. See [#467](https://github.com/eclipse/lsp4mp/pull/467).
+- fix: prevent from duplicate properties. See [#465](https://github.com/eclipse/lsp4mp/pull/465).
```

Comment:
"Code assist should avoid suggesting properties that already exist in document."

Reply:
It is not exactly that, the fix is when you compute properties from Java classpath, it generates before the PRduplicate properties.

Reply:
I rename the issue with `Prevent duplicate properties when generating them`

Reply:
fixed, but if you don't like my rename, please tell me.